napari/_vispy/layers/points.py

```diff
@@ -136,6 +137,9 @@ def _on_highlight_change(self):
             pos = self.layer._highlight_box
             width = scaled_highlight
 
+        # FIXME: vispy bug? LineVisual error when going from 2d to 3d (or the opposite)
```

Comment:
@JoOkuma I'm going through to split up points changes into their own PR - is this line part of a bad merge or did you add this? If you added it, can you expand a bit more what the bug is? Maybe this should be it's own tiny PR